### ISSUE TITLE
System.Xml.RW.XmlReader.Tests fails on iOS/tvOS

### ISSUE THREAD

**Issue by mdh1418** [dotnet/runtime#51391]
System.Xml.RW.XmlReader.Tests
System.Xml.Tests.AsyncReaderLateInitTests.InitializationWithUriOnNonAsyncReaderThrows
System.Xml.Tests.AsyncReaderLateInitTests.ReadAfterInitializationWithUriOnAsyncReaderTrows
System.Xml.Tests.AsyncReaderLateInitTests.ReadAsyncAfterInitializationWithUriThrows

```
System.Xml.RW.XmlReader.Tests.dll   Failed: 3

Test collection for System.Xml.Tests.AsyncReaderLateInitTests
System.Xml.Tests.AsyncReaderLateInitTests.InitializationWithUriOnNonAsyncReaderThrows
    Assert.Throws() Failure\nExpected: typeof(System.Net.Http.HttpRequestException)\nActual:   typeof(System.PlatformNotSupportedException): System.Net.Quic is not supported on this platform.\n---- System.PlatformNotSupportedException : System.Net.Quic is not supported on this platform.
System.Xml.Tests.AsyncReaderLateInitTests.ReadAfterInitializationWithUriOnAsyncReaderTrows
    Assert.Throws() Failure\nExpected: typeof(System.Net.Http.HttpRequestException)\nActual:   typeof(System.PlatformNotSupportedException): System.Net.Quic is not supported on this platform.\n---- System.PlatformNotSupportedException : System.Net.Quic is not supported on this platform.
System.Xml.Tests.AsyncReaderLateInitTests.ReadAsyncAfterInitializationWithUriThrows
    Assert.Throws() Failure\nExpected: typeof(System.Net.Http.HttpRequestException)\nActual:   typeof(System.PlatformNotSupportedException): System.Net.Quic is not supported on this platform.\n---- System.PlatformNotSupportedException : System.Net.Quic is not supported on this platform.

**Comment by filipnavara** [comment]
This should be fixed by #49261.

**Comment by mdh1418** [comment]
Can confirm, they pass locally now :)
